pydax/_high_level.py

```diff
@@ -185,9 +185,11 @@ def load_dataset(name: str, *,
     2 2010-01-01 03:00:00               5.0                33.0
     """
 
-    schema = export_schemata().schemata['datasets'].export_schema('datasets', name, version)
+    schemata = export_schemata().schemata['datasets']
+    schema = schemata.export_schema('datasets', name, version)
+    schema_name = schemata.export_schema().get('name', 'default')
 
-    data_dir = get_config().DATADIR / name / version
+    data_dir = get_config().DATADIR / schema_name / name / version
```

Comment:
Can we add the same change to the `Dataset` doctest? https://github.com/CODAIT/pydax/blob/master/pydax/_dataset.py#L56

Reply:
Also in `test_dataset`: https://github.com/CODAIT/pydax/blob/master/tests/test_dataset.py#L260 for consistency

Reply:
I updated the dataset doctest and the test so that they no longer follow some directory structure. They are not high-level functions and they shouldn't be required to follow these directory structures (and when users use them, they are more likely to not follow the high-level directory structures)